dashboard/app/models/school_info.rb

```diff
@@ -32,7 +32,8 @@ class SchoolInfo < ApplicationRecord
     SCHOOL_TYPE_HOMESCHOOL = "homeschool".freeze,
     SCHOOL_TYPE_AFTER_SCHOOL = "afterschool".freeze,
     SCHOOL_TYPE_ORGANIZATION = "organization".freeze,
-    SCHOOL_TYPE_OTHER = "other".freeze
+    SCHOOL_TYPE_OTHER = "other".freeze,
+    SCHOOL_TYPE_NO_SCHOOL_SETTING = "noSchoolSetting".freeze
```

Comment:
here's the new school type for non-school settings

Reply:
tiny tiny nit: should this be before other? I feel like "other" is always at the end of these lists.

Not worth updating if there aren't other changes to make. And definitely not blocking either way